server/channels/app/platform/helper_test.go

```diff
@@ -32,6 +33,7 @@ type TestHelper struct {
 	// BasicPost    *model.Post
 
 	SystemAdminUser *model.User
+	Tb              testing.TB
```

Comment:
You also need to assign something to the field :wink:

Reply:
This comment is still unresovled

Reply:
Hey @hanzei  not sure what you meant there, assign what? and at what point? Tried looking in the other helper files for an example but couldn't see any assigning